projects/fmcomms8/zcu102/README.md

```diff
@@ -0,0 +1,18 @@
+# FMCOMMS8/ZCU102 HDL Project
+
+## Building the project
+
+```
+cd projects/fmcomms8/zcu102
+make
+```
+
+All of the RX/TX link modes can be found in the [AD9081 data sheet](https://www.analog.com/media/en/technical-documentation/user-guides/ad9081-ad9082-ug-1578.pdf). We offer support for only a few of them.
+
+The overwritable parameters from the environment:
+
+- [RX/TX]_JESD_M - [RX/TX] number of converters per link
+- [RX/TX]_JESD_L - [RX/TX] number of lanes per link
+- [RX/TX]_JESD_S - [RX/TX] number of samples per converter per frame
```

Comment:
You should mention the RX_OS (RX observation path) ones as well https://github.com/analogdevicesinc/hdl/blob/main/projects/fmcomms8/zcu102/system_project.tcl#L28-L36